SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -11,6 +11,10 @@ import PushKit
 import TwilioVoice
 
 let baseURLString = <#URL TO YOUR ACCESS TOKEN SERVER#>
+/**
+ If your token server is written in PHP, accessTokenEndpoint needs .php extension at the end.
```

Comment:
Format nitpick:
```
/**
 * block comment
 */
```

Reply:
Was referring to http://nshipster.com/swift-documentation/  :) .. will fix.

Reply:
And https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/ComentBlock.html